sdk/enter.go

```diff
@@ -182,17 +182,19 @@ func (e *enter) mountGnupg() error {
 		origHome = filepath.Join(e.User.HomeDir, ".gnupg")
 	}
 
-	if _, err := os.Stat(origHome); err != nil {
-		// Skip but do not bind mount anything
-		return nil
-	}
-
 	// gpg misbehaves in the sdk with GNUPGHOME set to anything but ~/.gnupg
 	// so always unset it so the default ~/.gnupg is used.
+	// This is no longer in the env_keep setup for sudo when creating the
+	// sdk, but that won't work with existing sdk's.
```

Comment:
`that won't work with` doesn't parse cleanly for me.  What about:

> gpg misbehaves in the SDK with GNUPGHOME set to anything but ~/.gnupg.  New SDKs don't encounter this problem because GNUPGHOME is no longer included in the sudo env_keep list.  For old SDKs, unset GNUPGHOME so the default ~/.gnupg is used.